app/src/main/java/com/babylon/wallet/android/presentation/account/history/HistoryScreen.kt

```diff
@@ -210,28 +210,27 @@ fun HistoryContent(
     )
     LaunchedEffect(state.timeFilterItems.size) {
         if (state.timeFilterItems.isNotEmpty()) {
-            Timber.d("History: Scrolling to last item")
             timeFilterScrollState.scrollToItem(state.timeFilterItems.lastIndex)
         }
     }
     MonitorListScroll(state = listState, onLoadMore = { direction ->
         when (direction) {
             ScrollInfo.Direction.UP -> {
-                if (state.canLoadMoreUp) {
-                    onLoadMore(ScrollInfo.Direction.UP)
-                }
+                onLoadMore(ScrollInfo.Direction.UP)
```

Comment:
`state` in previous version was not properly refreshed here since it was not a part of key of LaunchedEffect that triggered `onLoadMore` callback. This bug led to load more callback not firing up properly in some circumstances